content/python/concepts/built-in-functions/terms/print/print.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'print()'
+Description: 'Takes in data such as a string or an int converted to string using str() and prints it to the standard output device.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The print() function takes in a string or any other element such as int converted to a string using str() and displays those values to the standard output device.
```

Comment:
```suggestion
The `print()` function takes in a string or any other element such as an integer converted to a string using `str()` and displays those values to the standard output device.
```

I get what this is trying to say 👍 however, it's not very clear at first glance. The `print()` function implicitly calls on `str()`. This is called type casting or type conversion and we have on Docs entry on that: [Python casting](https://www.codecademy.com/resources/docs/python/casting)! I suggest linking out to this source.

Let's re-write this sentence to explain that the `print()` function takes in an argument and displays it to the "standard output device (as you've stated)." Then, explain how the `print()` function works.